samples/drawing/drawing.cpp

```diff
@@ -1834,7 +1836,7 @@ void MyCanvas::OnPaint(wxPaintEvent &WXUNUSED(event))
 void MyCanvas::Draw(wxDC& pdc)
 {
 #if wxUSE_GRAPHICS_CONTEXT
-    wxGCDC gdc;
+    wxGCDC gdc(this);
```

Comment:
Should we add a public `wxGCDC::SetWindow()` too? AFAICS there is no way to change it later if the default ctor is used.

Or maybe we could inherit the scale factor from the context when `SetGraphicsContext()` is called?

Reply:
We could probably inherit it from `wxGraphicsContext`. Since `wxDC` doesn't have `SetWindow()` I don't think `wxGCDC` should have it.